### PR TITLE
feat: throw InvalidArgument instead of UnsupportedOperation for print…

### PR DESCRIPTION
**Pull request by thiagowfx**
…ing an empty content area

Related, in sync with, and blocked on: https://github.com/web-platform-tests/wpt/pull/40872

### CODE REVIEW

**Comment by thiagowfx** [comment]
This PR should not be submitted until the WPT test above is merged.

**Comment by thiagowfx** [comment]
I'm mistaken, it turns out this isn't blocked on upstream. It's actually the other way around: this needs to be landed before the WPT test.

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**
